.phpstan/src/Type/ImageStorageGetNoImageMethodReturnTypeExtension.php

```diff
@@ -0,0 +1,61 @@
+<?php declare(strict_types = 1);
+
+namespace Contributte\ImageStorage\PHPStan\Type;
+
+use Contributte\ImageStorage\Image;
+use Contributte\ImageStorage\ImageNameScript;
+use Contributte\ImageStorage\ImageStorage;
+use PhpParser\Node\Expr\MethodCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\MethodReflection;
+use PHPStan\Reflection\ParametersAcceptorSelector;
+use PHPStan\Type\Constant\ConstantArrayType;
+use PHPStan\Type\Constant\ConstantBooleanType;
+use PHPStan\Type\DynamicMethodReturnTypeExtension;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\StringType;
+use PHPStan\Type\Type;
+
+final class ImageStorageGetNoImageMethodReturnTypeExtension implements DynamicMethodReturnTypeExtension
```

Comment:
Uh, thank you. But I think it's not needed. We don't create custom for phpstan extensions for single package. I mean, if we do, we have to do it for all packages to be consistence across all packages.

Reply:
This is needed for level 5+, because this method returns union. Previously it had php typehint for 1 type, which should cause type error from php.

Reply:
btw users of this library can include this extension in their phpstan build and benefit from it too - less false positive/no ignore error because of union

Reply:
Thanks a lot for the PR but I would also consider removing this extension as it would require more maintenance. Thx

Reply:
as you wish